docs/experiment/ppo.rst

```diff
@@ -47,6 +48,8 @@ NVIDIA GPUs
 +----------------------------------+------------------------+------------+-----------------------------------------------------------------------------------------------+
 | Qwen/Qwen2-7B-Instruct           | GRPO                   | 89         |   `Qwen7b GRPO Script`_                                                                       |
 +----------------------------------+------------------------+------------+-----------------------------------------------------------------------------------------------+
+| Qwen/Qwen2-7B-Instruct           | GRPO (FSDP2)           | 89.8       |  `_Qwen7b GRPO FSDP2 Script and Logs`_                                                                       |
++----------------------------------+------------------------+------------+-----------------------------------------------------------------------------------------------+
```

Comment:
use the same number of `-` to make the table have correct render and also remove `_` in `_Qwen7b GRPO FSDP2 Script and Logs` for indexing.